masterlist.yaml

```diff
@@ -5973,6 +5973,12 @@ plugins:
       - crc: 0xA4E835CA
         util: 'SSEEdit v4.0.3'
 
+  - name: 'Cathedral - 3D Pine Grass.esp'
+    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/42032/' ]
+    after:
+      - 'Cathedral Landscapes.esp'
```

Comment:
LOOT sorts `Cathedral - 3D Pine Grass.esp` after `Cathedral Landscapes.esp` by default, but it wouldn't hurt to add a rule anyway if you still want to.

Reply:
Hmm. It seems to work that way. If there is an implicit rule that sorts `Cathedral - 3D Pine Grass.esp` after `Cathedral Landscapes.esp`, I will cancel the pull request.

Reply:
If you look in the file header for `Cathedral - 3D Pine Grass.esp`, you'll see its masters.
Plugins will always load after their master files.

Like below.
![image](https://user-images.githubusercontent.com/1944639/111557308-523b8f80-8784-11eb-93f4-18eece0e21d5.png)

Reply:
There is no dependency between those two mods, that is defined in the header. I was asking if there was a hidden rule that somehow sorted the load order correctly.

Reply:
> There is no dependency between those two mods, that is defined in the header. I was asking if there was a hidden rule that somehow sorted the load order correctly.

No there's not a hidden rule, but plugins are ordered using a sorting algorithm.
The sorting algorithm determined it was the optimal order.

Reply:
@sean-kang Sorry for the confusion! When I said it sorted after by default, I was referring to the sorting algorithm Mac mentioned.